project/src/components/DetailsNavigation/index.tsx

```diff
@@ -9,20 +10,33 @@ type DetailsNavigationProps = {
   film: Film;
 }
 
+
 export const DetailsNavigation = ({ film }:DetailsNavigationProps) => {
-  const [currentButton, setCurrentButton] = useState<NavigationButtonsType>(NAVIGATION_BUTTONS.DETAILS);
+
+  const {id} = useParams();
+  const {currentInformation} = useParams();
+  const navigate = useNavigate();
+
+  const [, setCurrentButton] = useState<NavigationButtonsType>(currentInformation as NavigationButtonsType);
+
 
   const handlerActiveButton = (evt: React.MouseEvent<HTMLButtonElement>) => {
     const target = evt.target as HTMLButtonElement;
     const buttonName = target.getAttribute('data-name');
 
-    if (buttonName) {
+    if (buttonName && id) {
       if (
         Object
           .values(NAVIGATION_BUTTONS)
           .includes(buttonName as NavigationButtonsType)
       ) {
         setCurrentButton(buttonName as NavigationButtonsType);
+
+        const path = generatePath('/films/:id/:currentInformation', {
```

Comment:
let's store it in global constants `/films/:id/:currentInformation`

Reply:
fixed